### PR TITLE
Re-fix insulation jank. Give lizards homeostasis

### PR DESCRIPTION
**Pull request by thegrb93**
## About The Pull Request
Apparently we want lizards to survive outside in icebox with passive winter clothes despite not having the ability to regulate body temperature. They were able to do this before since insulation code was janked and prevented their temp from reaching the outside temperature.

This PR gives lizards some homeostasis so they can survive with passive winter clothes in the cold of icebox and fixes the insulation jank again.
## Why It's Good For The Game
Insulation works correctly again and lizards don't freeze to death while wearing winter clothes.
## Changelog
:cl:
fix: Fixes insulation jank again
balance: Gives lizards weak bodytemp regulation so they can survive cold environments with passive insulation.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/_species.dm
Comment (SmArtKar) [inline code/modules/mob/living/carbon/human/_species.dm:1223]: You are just reintroducing the body temp jank I removed specifically due to buggy implementation while adding temperature regulation to lizards. This is a species change without maintainer approval which is required for things like these. Its not a fix, its balance too.

**Review by SmArtKar — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/_species.dm
Comment (thegrb93) [reply code/modules/mob/living/carbon/human/_species.dm:1223]: I explained in my last PR that the original code is wrong and causes the body temp to get stuck. What 'jank' are you referring to with my fix?

Comment (thegrb93) [reply code/modules/mob/living/carbon/human/_species.dm:1223]: Fixed the changelog

Comment (Sadboysuss) [reply code/modules/mob/living/carbon/human/_species.dm:1223]: > species change without maintainer approval

Species changes need headcoder approval, not maintainer.

**Comment by Watermelon914** [comment]
We require maintainer approval whenever someone makes a species change.

I'm not entirely convinced that any of the code you've touched causes any sort of bug or problems. It's intentional that lizards are not able to regulate their body temperature. If they find themselves unable to survive in partially insulated clothing, that's a them issue and you can consider adding spacesuits that allow body temp regulation on icebox to compensate for the disadvantage lizards have.

**Comment by thegrb93** [comment]
@Watermelon914 someone needs to do something because this issue is back now. https://github.com/NovaSector/NovaSector/issues/201

**Comment by Sadboysuss** [comment]
> @Watermelon914 someone needs to do something because this issue is back now. [NovaSector/NovaSector#201](https://github.com/NovaSector/NovaSector/issues/201)

it's not a bug, you're welcome to make an issue report upstream here to see if another issue manager or maint has a different opinion.

**Comment by BlueMemesauce** [comment]
> We require maintainer approval whenever someone makes a species change.
> 
> I'm not entirely convinced that any of the code you've touched causes any sort of bug or problems. It's intentional that lizards are not able to regulate their body temperature. If they find themselves unable to survive in partially insulated clothing, that's a them issue and you can consider adding spacesuits that allow body temp regulation on icebox to compensate for the disadvantage lizards have.

He just made a pr where he made it where lizards were unable to regulate their body temperature and couldn't survive in icebox, and that was reverted because lizards shouldn't die in icebox. So he made this as an alternative and then you're telling him to go back to what he previously did and was reverted.  What is he supposed to do in this situation?

**Pull request closed without merging**
